### PR TITLE
Update Idv docs

### PR DESCRIPTION
**Pull request by divyaamunugama**

### CODE REVIEW

File: en/docs/guides/identity-verification/identity-verification-overview.md [hunk 1]
```diff
@@ -11,18 +11,15 @@ themselves from potential threats and enhance the overall security of their syst
...
-You can use the [identity verification management API](../../apis/idv-rest-api.md) to add, update and delete identity verification providers
+You can use the [identity verification management API]({{base_path}}/apis/idv-rest-api.md) to add, update and delete identity verification providers
 
 ## Configuring an Identity Verification Provider
 
-An identity verification provider (IDVP) is a third-party service that is used to verify the identity of a user. 
-Through Identity Server, you can configure an IDVP to verify the identity of a user. In order to onboard IDVP connectors,
-you have to deploy the required IDVP connectors from the connector store.
-
-In order to onboard an IDVP to WSO2 Identity Server, you need to deploy the identity verification connector artifacts.
```
Comment (divyaamunugama) [inline en/docs/guides/identity-verification/identity-verification-overview.md:22]: Removed this section, as we are not providing any information on how to download it from the connector store. Hope this is okay.

File: en/mkdocs.yml [hunk 2]
```diff
@@ -612,9 +612,9 @@ nav:
           - JIT Consent Purposes: guides/identity-federation/resident-jit-consent.md
       - Identity Verification:
         - Overview: guides/identity-verification/identity-verification-overview.md
-        - Register an Identity Verification Provider: guides/identity-verification/add-idvp.md
-        - Manage an Identity Verification Provider: guides/identity-verification/manage-idvp.md
-        - Onboard the UI of an Identity Verification Provider: guides/identity-verification/idvp-ui.md
+        - Register an IdVP: guides/identity-verification/add-idvp.md
+        - Manage an IdVP: guides/identity-verification/manage-idvp.md
+        - Onboard the UI of an IdVP: guides/identity-verification/idvp-ui.md
```
Comment (divyaamunugama) [inline en/mkdocs.yml:617]: hope its okay to use `IdVP`

File: en/docs/guides/identity-verification/idvp-ui.md [hunk 3]
```diff
@@ -105,207 +104,209 @@ Refer [validation.ts](https://github.com/wso2/identity-apps/blob/master/modules/
...
+``` json
 {
-    "Type": "< IDV Provider type >",
-    "Name": "< A default name for the IDV Provider >",
-    "description": "< A default description for the IDV Provider >",
-    "isEnabled": < Should the IDV Provider be enabled by default >,
+    "Type": "<IDV Provider type>",
+    "Name": "<A default name for the IDV Provider>",
+    "description": "<A default description for the IDV Provider>",
+    "isEnabled": <Should the IDV Provider be enabled by default>,
```
Comment (divyaamunugama) [inline en/docs/guides/identity-verification/idvp-ui.md:260]: shouldn't `<Should the IDV Provider be enabled by default>` be within inverted comas/quotes?

File: en/docs/guides/identity-verification/idvp-ui.md [hunk 4]
```diff
@@ -105,207 +104,209 @@ Refer [validation.ts](https://github.com/wso2/identity-apps/blob/master/modules/
...
         .... other default claim mappings ...
     ],
     "configProperties": [
     {
-        "key": "< key of the config property >",
-        "value": "< default value for the config property >",
-        "isSecret": < should the property be stored as a secret  >
+        "key": "<key of the config property>",
+        "value": "<default value for the config property>",
+        "isSecret": <should the property be stored as a secret>
```
Comment (divyaamunugama) [inline en/docs/guides/identity-verification/idvp-ui.md:272]: same comment as above

File: en/mkdocs.yml (hunk 2 above)
Comment (GANGANI) [reply en/mkdocs.yml:617]: +1

File: en/docs/guides/identity-verification/idvp-ui.md (hunk 3 above)
Comment (GANGANI) [reply en/docs/guides/identity-verification/idvp-ui.md:260]: yes. It should

**Review by GANGANI — APPROVED** [review]

**Review by nipunsampath — APPROVED** [review]

**Pull request merged**